client/src/related/relatedList.jsx

```diff
@@ -31,8 +38,16 @@ class RelatedList extends React.Component {
         item: this.props.item
```

Comment:
Does this need a declaration?

Reply:
I am also getting this '@@ -31,8 +38,16 @@ class RelatedList extends React.Component {' greyed out in the old file.  I have never it before but if it all looks good, then please ignore me!